spring-cloud-stream-reactive/src/test/java/org/springframework/cloud/stream/reactive/StreamListenerReactorTests.java

```diff
@@ -165,7 +334,17 @@ public void receive(@Input(Processor.INPUT) Flux<String> input, @Output(Processo
 
 	@EnableBinding(Processor.class)
 	@EnableAutoConfiguration
-	public static class TestInputOutputArgsWithMessage {
+	public static class TestInputOutputArgs2 {
+
+		@StreamListener(Processor.INPUT)
```

Comment:
On a second thought, I think it would be a convenient for the users to specify the binding targets in whatever places that it actually matters. For instance, user knows either `@StreamListener` or the method parameter with `@Input` accepts the inbound target and they can specify wherever. We already cover some checks if they specify in both the places etc., This way, we don't have to restrict the user to follow a specific path to specify the targets while those way of specifying would still look correct logically.  What do you think?